### ISSUE TITLE
Can not run OPS-SAT simulation environment

### ISSUE THREAD

**Issue by fuchs257** [esa/nanosat-mo-framework#50]
Hello,
I pulled latest changes of nanosat-mo-framework and nmf-mission-ops-sat and installed them with maven (nmf-mission with -Pground).
The problem is I can not get the simulation environment running.

Starting the ground-mo-proxy gives me info 
`INFO: Ground MO Proxy initialized! URI: maltcp://172.17.0.1:1025/ground-mo-proxy-Directory`
so I think thats fine.
starting the supervisor-sim however gives me a NullPointerException:

```
Dec 09, 2020 11:44:36 AM esa.mo.sm.impl.provider.PackageManagementProviderServiceImpl init
INFO: Package Management service READY
Dec 09, 2020 11:44:36 AM esa.mo.com.impl.provider.ArchiveSyncProviderServiceImpl init
INFO: ArchiveSync service READY
Dec 09, 2020 11:44:36 AM esa.mo.mc.impl.provider.ParameterProviderServiceImpl init
INFO: Parameter service READY
Dec 09, 2020 11:44:36 AM esa.mo.mc.impl.provider.ActionProviderServiceImpl init
INFO: Action service READY
Dec 09, 2020 11:44:36 AM esa.mo.mc.impl.provider.AlertProviderServiceImpl init
INFO: Alert service READY
Dec 09, 2020 11:44:36 AM esa.mo.mc.impl.provider.AggregationProviderServiceImpl init
INFO: Aggregation service READY
OPS-SAT SoftSim:Sim;INFO;opssat.simulator.threading.TaskNode;initLogging;File logging level is [INFO], Console logging level is [INFO]
Dec 09, 2020 11:44:36 AM esa.mo.nmf.nanosatmosupervisor.NanosatMOSupervisorBasicImpl initPlatformServices
SEVERE: Something went wrong when initializing the platform services. Using simulated services.
java.lang.NullPointerException
        at opssat.simulator.util.GenericFileBasedOperatingBuffer.loadImageFromAbsolutePath(GenericFileBasedOperatingBuffer.java:71)
        at opssat.simulator.threading.SimulatorNode.initModels(SimulatorNode.java:426)
        at opssat.simulator.threading.SimulatorNode.<init>(SimulatorNode.java:556)
        at opssat.simulator.main.ESASimulator.<init>(ESASimulator.java:144)
        at esa.mo.platform.impl.util.PlatformServicesProviderSoftSim.<init>(PlatformServicesProviderSoftSim.java:61)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
        at java.lang.Class.newInstance(Class.java:442)
        at esa.mo.nmf.nanosatmosupervisor.NanosatMOSupervisorBasicImpl.initPlatformServices(NanosatMOSupervisorBasicImpl.java:40)
        at esa.mo.nmf.nanosatmosupervisor.NanoSatMOSupervisor.init(NanoSatMOSupervisor.java:105)
        at esa.mo.nmf.nanosatmosupervisor.NanosatMOSupervisorBasicImpl.init(NanosatMOSupervisorBasicImpl.java:67)
        at esa.mo.nmf.nanosatmosupervisor.NanosatMOSupervisorBasicImpl.main(NanosatMOSupervisorBasicImpl.java:80)

OPS-SAT SoftSim:Sim;INFO;opssat.simulator.threading.TaskNode;initLogging;File logging level is [INFO], Console logging level is [INFO]
OPS-SAT SoftSim:Sim;INFO;opssat.simulator.threading.TaskNode;initLogging;File logging level is [INFO], Console logging level is [INFO]
Exception in thread "main" java.lang.NullPointerException
        at opssat.simulator.util.GenericFileBasedOperatingBuffer.loadImageFromAbsolutePath(GenericFileBasedOperatingBuffer.java:71)
        at opssat.simulator.threading.SimulatorNode.initModels(SimulatorNode.java:426)
        at opssat.simulator.threading.SimulatorNode.<init>(SimulatorNode.java:556)
        at opssat.simulator.main.ESASimulator.<init>(ESASimulator.java:144)
        at esa.mo.platform.impl.util.PlatformServicesProviderSoftSim.<init>(PlatformServicesProviderSoftSim.java:61)
        at esa.mo.nmf.nanosatmosupervisor.NanosatMOSupervisorBasicImpl.initPlatformServices(NanosatMOSupervisorBasicImpl.java:48)
        at esa.mo.nmf.nanosatmosupervisor.NanoSatMOSupervisor.init(NanoSatMOSupervisor.java:105)
        at esa.mo.nmf.nanosatmosupervisor.NanosatMOSupervisorBasicImpl.init(NanosatMOSupervisorBasicImpl.java:67)
        at esa.mo.nmf.nanosatmosupervisor.NanosatMOSupervisorBasicImpl.main(NanosatMOSupervisorBasicImpl.java:80)

```

Do I have to configure something ?

hope someone can help
Cheers,
Marius

**Comment by dmarszk** [comment]
@fuchs257 please try now - it should be fixed.

**Comment by fuchs257** [comment]
The errors before are gone, but now I get a new one
```
INFO: Aggregation service READY
OPS-SAT SoftSim:PRE_INIT: Header was not found, a default header will be created.
OPS-SAT SoftSim:Sim;INFO;opssat.simulator.threading.TaskNode;initLogging;File logging level is [INFO], Console logging level is [INFO]
OPS-SAT SoftSim:Sim;WARNING;opssat.simulator.threading.SimulatorNode;loadTemplatesFromFile;Templates file [_OPS-SAT-SIMULATOR-templates.txt] not found!
OPS-SAT SoftSim:Sim;WARNING;opssat.simulator.threading.SimulatorNode;loadSchedulerFromFile;Scheduler file [_OPS-SAT-SIMULATOR-scheduler.txt] not found!
OPS-SAT SoftSim:Sim;SEVERE;opssat.simulator.threading.SimulatorNode;initModels;orekit initialization failed from [org.orekit.errors.OrekitException: no JPL ephemerides binary files found]! Switching module off
OPS-SAT SoftSim:Cen;INFO;opssat.simulator.threading.TaskNode;initLogging;File logging level is [INFO], Console logging level is [INFO]
OPS-SAT SoftSim:Cen;INFO;opssat.simulator.tcp.MultiThreadedSocketServer;run;ServerSocket created on port [11111]
OPS-SAT SoftSim:Cen;INFO;opssat.simulator.celestia.CelestiaIf;openConnection;Created ServerSocket for Celestia on port [5909]
Dec 10, 2020 7:00:01 AM esa.mo.platform.impl.provider.opssat.CameraOPSSATAdapter <init>
INFO: Initialisation
Exception in thread "main" java.lang.UnsatisfiedLinkError: no ims100_api_jni in java.library.path
        at java.lang.ClassLoader.loadLibrary(Unknown Source)
        at java.lang.Runtime.loadLibrary0(Unknown Source)
        at java.lang.System.loadLibrary(Unknown Source)
        at esa.mo.platform.impl.provider.opssat.CameraOPSSATAdapter.<init>(CameraOPSSATAdapter.java:86)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
        at java.lang.reflect.Constructor.newInstance(Unknown Source)
        at java.lang.Class.newInstance(Unknown Source)
        at esa.mo.platform.impl.util.PlatformServicesProviderSoftSim.init(PlatformServicesProviderSoftSim.java:92)
        at esa.mo.nmf.nanosatmosupervisor.NanosatMOSupervisorBasicImpl.initPlatformServices(NanosatMOSupervisorBasicImpl.java:51)
        at esa.mo.nmf.nanosatmosupervisor.NanoSatMOSupervisor.init(NanoSatMOSupervisor.java:105)
        at esa.mo.nmf.nanosatmosupervisor.NanosatMOSupervisorBasicImpl.init(NanosatMOSupervisorBasicImpl.java:67)
        at esa.mo.nmf.nanosatmosupervisor.NanosatMOSupervisorBasicImpl.main(NanosatMOSupervisorBasicImpl.java:80)

```

**Comment by fuchs257** [comment]
fixed in nmf-mission-opssat 2cfa840
thanks
